pkg/scheduler/schedulercache/cache.go

```diff
@@ -277,14 +277,14 @@ func (cache *schedulerCache) UpdatePod(oldPod, newPod *v1.Pod) error {
 	// before Update event, in which case the state would change from Assumed to Added.
 	case ok && !cache.assumedPods[key]:
 		if currState.pod.Spec.NodeName != newPod.Spec.NodeName {
-			glog.Errorf("Pod %v updated on a different node than previously added to.", key)
+			glog.Errorf("pod %v updated on a different node than previously added to.", key)
```

Comment:
start with big letter for log

Reply:
You may want to say upper case, haha

Reply:
Just to be clear, @k82cn, this is right, it should start with lower case?